pyfilesync/tests/__init__.py

```diff
@@ -1,17 +1,15 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
+
```

Comment:
Zbędna linia, z nią może moduł nie zaczytać dokumentacji (ale to można sprawdzić)

Reply:
Miałem na myśli tą linię przed komentarzem ;)